06-building-a-program-start-to-finish/writing-tests/Output/test/spec/test.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
```

Comment:
The name `test.js` communicates little about what this file does.

Following convention this would look like `inverted-index.spec.js`